homeassistant/components/shelly/event.py

```diff
@@ -68,6 +69,12 @@ class ShellyRpcEventDescription(EventEntityDescription):
         config, status, key
     ),
 )
+SCRIPT_EVENT: Final = ShellyRpcEventDescription(
+    key="script",
+    translation_key="script",
+    device_class=None,
+    entity_registry_enabled_default=False,
```

Comment:
We want to use the device name in the entity name.
```suggestion
    entity_registry_enabled_default=False,
    has_entity_name=True,
```